CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## dev
+
+* `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was
```

Comment:
ideally, we'd use the tag descriptions for these things. see this [Releases](https://github.com/tweenjs/tween.js/releases) page for example.